easydiffraction/Job.py

```diff
@@ -59,6 +60,8 @@
 except ImportError:
     print("pandas not installed")
 
+if 'JPY_PARENT_PID' in os.environ:
```

Comment:
It would probably be good to add a comment that this means 'if the job is run from the notebook', as it isn't obvious

Reply:
Added a method to determine if code is running in Jupyter notebook.